magna/api/urls.py~

```diff
@@ -0,0 +1,31 @@
+from django.conf.urls import url
```

Comment:
the files that end with ~ are copies of files local on your machine. prob should not be committed